index.js

```diff
@@ -76,13 +82,76 @@ if (process.platform === 'win32') {
 
 getFn = function () {
 	var fn = function (/*…msg*/) {
-		var start = '', end = '';
-		forEach(fn._cliColorData, function (mod) {
-			end = '\x1b[' + mod[1] + 'm' + end;
-			start += '\x1b[' + mod[0] + 'm';
-		}, null, true);
-		return start + join.call(arguments, ' ') + end;
+		var start = ''
+		  , end   = ''
+		  , keys  = Object.keys(fn._cliColorData)
+		  , msg   = join.call(arguments, ' ');
+
+		if (keys.length) {
+			for (var i=0; i<keys.length; i++) {
+				start += ( i !== 0 ? ';' : '' ) + fn._cliColorData[keys[i]][0];
+				end   += ( i !== 0 ? ';' : '' ) + fn._cliColorData[keys[i]][1];
+			}
+
+			start = '\x1b[' + start + 'm';
+			end   = '\x1b[' + end + 'm';
```

Comment:
I changed it because I first need to check if `_cliColorData` is a not empty object. So I can do it by get `Object.keys()` of it and check by this `.length`. As I yet have the keys of data, so I can use directly `for`, without need run a callback on each data.

I did it because previously was generated control sequences separatedly, like `<blue><bgRed>`, now it is generated together (to help parser) like `<blue&bgReg>`.